### DISCUSSION TITLE
Design Notes - Footer

### DISCUSSION THREAD

**Discussion by dkh-uci** [naccdata/uniform-data-set#60] [Forms] [answered]
Hi All,
A quick question about Footer:
In the design notes for form creation, there is this instruction for Footers:

Footers
A Footer field should appear at the bottom of every form. Create a descriptive text box field with a variable name that follows the convention formname_footer (e.g. a3_footer). This field is uniform for all packets (telephone, in-person, etc).
The footer should pipe in the Form Title and Month Year.

![image](https://user-images.githubusercontent.com/101678603/216686521-94103cdc-04c3-41ec-b83d-2db0c32f947e.png)

**For [Month Year] where should the correct variable be piped from?**
I figured how to pipe in the form title as [instrument-label]. 
 
FYI:  This the field text I have for the footers, at least for the forms b5 & b8 that I'm working on:

<font color=#7591C0>National Alzheimer's Coordinating Center | (206) 543-8637 | fax: (206) 616-5927 | <a href="mailto:naccmail@uw.edu">naccmail@uw.edu</a> | </font color=#7591C0> naccdata.org
<b>UDS Version 4.0, <font color=red>[instrument-label], [Month Year]</font color=red></b> 

code below:
```
<font color=#7591C0>National Alzheimer's Coordinating Center | (206) 543-8637 | fax: (206) 616-5927 | <a href="mailto:naccmail@uw.edu">naccmail@uw.edu</a> | </font color=#7591C0> naccdata.org
<b>UDS Version 4.0, <font color=red>[instrument-label], [Month Year]</font color=red></b> 
```
![image](https://user-images.githubusercontent.com/101678603/216687433-6ec41c86-4afc-4bdc-9951-cb6bcbee79e6.png)

~Dan

**Comment by dkh-uci** [comment, marked as answer]
Going to use this for footers to align with the updated design notes for Footers:
https://nacc.redcap.rit.uw.edu/surveys/?__dashboard=79R93FFT39E

`<p><span style="color: #3598db;">NATIONAL ALZHEIMER'S COORDINATING CENTER | naccmail@uw.edu |</span> <span style="text-decoration: underline;"><a href="https://naccdata.org/" target="_blank" rel="noopener">naccdata.org</a></span><br /><strong>UDS <span style="font-weight: normal;">(Version 4.0, Month Year)</span> <span style="color: red;">[instrument-label]</span></strong></p>`

**Reply by SuzanneHunt-KC** [reply]
I was also going to ask about month and year in the footer.  I assumed that NACC wanted the month and year that UDS 4.0 "began".  The Development and Review document made it sound like the footer should have month/year of the visit, but that is not was has been done in the past.  Thanks for the tip on the smart variable [instrument-label]
